mpas_analysis/obs/observational_datasets.xml

```diff
@@ -0,0 +1,947 @@
+    </references>
+    <bibtex>
+      @misc{Yi2009,
+      author = {D. Yi and H. J. Zwally},
+      title = {Arctic Sea Ice Freeboard and Thickness, Version 1},
+      year = {2009},
+      doi = {10.5067/7Q8HCCWS4I0R},
+      url = {https://doi.org/10.5067/SXJVJ3A2XIZT}
+      }
+    </bibtex>
+    <dataUrls>
+      (missing, requires registration with NSIDC)
+    </dataUrls>
+    <preprocessing>
+    </preprocessing>
+    <tasks>
+      - timeSeriesSeaIceAreaVol
+    </tasks>
+  </observation>
+
+  <observation>
+    <name>
+      PIOMAS Arctic Sea Ice Volume Reanalysis
+    </name>
+    <component>
+      seaice
+    </component>
+    <description>
+      Sea Ice Volume is calculated using the Pan-Arctic Ice Ocean Modeling and
+      Assimilation System (PIOMAS, Zhang and Rothrock, 2003) developed at
+      APL/PSC.  Anomalies for each day are calculated relative to the average
+      over the 1979 -2016 period for that day of the year to remove the annual
+      cycle. The model mean annual cycle of sea ice volume over this period
+      ranges from 28,000 km3 in April to 11,500 km3 in September.
+    </description>
+    <source>
+      [PIOMAS website](http://psc.apl.uw.edu/research/projects/arctic-sea-ice-volume-anomaly/)
+    </source>
+    <releasePolicy>
+      Data is public, but they optionally ask for basic information about the
+      person downloading the data (name, e-mail, and affiliation).
+    </releasePolicy>
+    <references>
+      - [Schweiger et al. (2011)](https://doi.org/10.1029/2011JC007084)
+      - [Zhang and Rothrock (2003)](https://doi.org/10.1175/1520-0493%282003%29131&lt;0845:MGSIWA&gt;2.0.CO;2)
+    </references>
+    <bibtex>
+      @article{Schweiger2011,
+      author = {Axel Schweiger and Ron Lindsay and Jinlun Zhang and Mike Steele and Harry Stern and Ron Kwok},
+      title = {Uncertainty in modeled Arctic sea ice volume},
+      journal = {Journal of Geophysical Research: Oceans},
+      volume = {116},
+      number = {C8},
+      pages = {},
+      keywords = {Arctic, climate change, ice volume, modelling, sea ice},
+      doi = {10.1029/2011JC007084},
+      url = {https://agupubs.onlinelibrary.wiley.com/doi/abs/10.1029/2011JC007084},
+      year = {2011}
+      }
+      @article{Zhang2003,
+      author = { Jinlun  Zhang  and  D. A.  Rothrock },
+      title = {Modeling Global Sea Ice with a Thickness and Enthalpy Distribution Model in Generalized Curvilinear Coordinates},
+      journal = {Monthly Weather Review},
+      volume = {131},
+      number = {5},
+      pages = {845-861},
+      year = {2003},
+      doi = {10.1175/1520-0493(2003)131&lt;0845:MGSIWA&gt;2.0.CO;2},
+      URL = {https://doi.org/10.1175/1520-0493(2003)131&lt;0845:MGSIWA&gt;2.0.CO;2}
+      }
+    </bibtex>
+    <dataUrls>
+      - http://psc.apl.uw.edu/wordpress/wp-content/uploads/schweiger/ice_volume/PIOMAS.2sst.monthly.Current.v2.1.txt
+    </dataUrls>
+    <preprocessing>
+      (missing)
```

Comment:
@milenaveneziani, did you have a script for processing the text file into a NetCDF file?